src/main/java/core/basesyntax/service/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.strategy.impl.BalanceOperationService;
+import core.basesyntax.service.strategy.impl.PurchaseOperationService;
+import core.basesyntax.service.strategy.impl.ReturnOperationService;
+import core.basesyntax.service.strategy.impl.SupplyOperationService;
+import java.util.Map;
+
+public class OperationStrategy {
```

Comment:
Also I'd recommend creating a corresponding interface for this class and implement it here